wire-ios-sync-engine/Source/Synchronization/EventProcessor.swift

```diff
@@ -91,8 +91,10 @@ actor EventProcessor: UpdateEventProcessor {
     }
 
     private func enqueueTask(_ block: @escaping @Sendable () async throws -> Void) async throws {
+        defer { processingTask = nil }
```

Comment:
We need to nil out the processing task otherwise we'll keep re-throwing the first which occurs.